FWCore/Framework/src/EventProcessor.cc

```diff
@@ -1732,6 +1732,9 @@ namespace edm {
                             streamQueuesInserter_.push(*holder.group(), [this, status, holder, &es]() mutable {
                               for (unsigned int i = 0; i < preallocations_.numberOfStreams(); ++i) {
                                 streamQueues_[i].push(*holder.group(), [this, i, status, holder, &es]() mutable {
+                                  if (!status->setStreamStartedLumi(i)) {
```

Comment:
This would be replaced with `!status->shouldStreamStartLumi()`

Reply:
Done.